ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/fixture/CreatesParties.scala

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.sandbox.fixture
+
+import com.daml.ledger.api.auth.client.LedgerCallCredentials
+import com.daml.ledger.api.v1.admin.party_management_service.{
+  AllocatePartyRequest,
+  PartyManagementServiceGrpc,
+}
+
+import java.util.concurrent.Executors
+
+import scala.concurrent.duration.DurationInt
+import scala.concurrent.{Await, ExecutionContext, Future}
+
+trait CreatesParties {
```

Comment:
I believe eventually we will have to get rid of stackable trait pattern within our integration tests.